packages/react/src/Banner/Banner.features.stories.tsx

```diff
@@ -25,6 +25,7 @@ export const Critical = () => {
           to enable two-factor authentication as an additional security measure.
         </>
       }
+      onDismiss={action('onDismiss')}
```

Comment:
From a docs perspective we might not want this to be visible in the default story for critical, I think the idea was that it would be rare for critical banners to be dismissible since you must address them but I'm curious what @lukasoppermann would say 👀

Reply:
Agree, @joshblack and I discussed it and we would like to leave the the critical story as is, without the dismiss. Users can still get to it in the playground.

If you could to this change @brphelps, we can merge this. 

Thank you for this PR. 🙏

Reply:
@joshblack Alright, took care of that. Thanks!